docs/deploy/howto-install-iot-edge.md

```diff
@@ -1,18 +1,26 @@
-# How to Install Azure IoT Edge for Industrial Scenarios
+# How to Setup a Local Gateway for Industrial Scenarios
 
-[Home](readme.md)
```

Comment:
Please add this back - it is needed to navigate on the public GitHub.io documentation

Reply:
Done